src/envoy/mixer/README.md

```diff
@@ -38,7 +38,7 @@ This Proxy will use Envoy and talk to Mixer server.
 * Start Envoy proxy, run
 
 ```
-  bazel-bin/src/envoy/mixer/envoy -c src/envoy/mixer/envoy-mixer.conf
+  bazel-bin/src/envoy/mixer/start_envoy
```

Comment:
When I run this command, this file does not existed. Please help to fix. Thanks.